sweepbatcher/sweep_batcher.go

```diff
@@ -162,6 +162,21 @@ type VerifySchnorrSig func(pubKey *btcec.PublicKey, hash, sig []byte) error
 type FeeRateProvider func(ctx context.Context,
 	swapHash lntypes.Hash) (chainfee.SatPerKWeight, error)
 
+// InitialDelayProvider returns the duration after new batch creation before it
```

Comment:
nit: `InitialDelayProvider returns the duration after which a newly created batch is first published.`

Reply:
Fixed! Thanks!